File: rs2_dogadjaj_Ba/dogadjaj_ba/dogadjaj_ba.Services/Database/UserTicket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace dogadjaj_ba.Services.Database
{
    public class UserTicket
    {
        public int UserTicketID { get; set; }
        public int UserID { get; set; }
        public int TicketID { get; set; }

        public User User { get; set; }
        public Ticket Ticket { get; set; }
    }
}

File: mobile/lib/models/user_ticket.dart
import 'package:mobile/models/ticket.dart';
import 'package:mobile/models/user.dart';

class UserTicket {
  late int userTicketID;

  late User? user;
  late int? userID;
  late int? kolicina;

  late Ticket? ticket;
  late int? ticketID;
  

  UserTicket({
    required this.userTicketID,
    this.user,
    this.userID,
    this.ticket,
    this.ticketID,
    this.kolicina,
  });

  UserTicket.fromJson(Map<String, dynamic> json) {
    userTicketID = json['userTicketID'];
    userID = json['userID'];
    ticketID = json['ticketID'];
    kolicina = json['kolicina'];
    if (json['user'] != null) {
      user = User.fromJson(json['user']);
    } else {
      user = null;
    }
    if (json['ticket'] != null) {
      ticket = Ticket.fromJson(json['ticket']);
    } else {
      user = null;
    }

  }

  Map<String, dynamic> toJson() {
    final Map<String, dynamic> data = <String, dynamic>{};

    return data;
  }
}
